tests/conftest.py

```diff
@@ -10,6 +11,7 @@ def assert_print_op(
     expected: str,
     diagnostic: Diagnostic | None,
     print_generic_format: bool = True,
+    **printer_options: Any,
```

Comment:
why variadic here?

Reply:
This allows us to specify as many arguments as we want, as long as they are keyword arguments.